build.gradle

```diff
@@ -8,22 +10,20 @@ enum VersionType {
 
 ext {
     graalRevision = getGraalRevision()
-    project.logger.info("Graal revision: $graalRevision")
 
-    if (version.endsWith("-SNAPSHOT")) {
+    if (conclave_graal_version.endsWith("-SNAPSHOT")) {
         versionType = VersionType.SNAPSHOT
-    } else if (version.matches(/.+\-RC[0-9]+$/)) {
+    } else if (conclave_graal_version.matches(/.+\-RC[0-9]+$/)) {
```

Comment:
Let's use `version` as that makes it clearer we're dealing with the project version